["android.intent.action.MAIN", "android.intent.action.VIEW", "cn.jpush.android.ui.PushActivity", "com.hsbxjj.middleman.app.updateservice", "cn.jpush.android.intent.REGISTER", "cn.jpush.android.intent.REPORT", "cn.jpush.android.intent.PushService", "cn.jpush.android.intent.PUSH_TIME", "cn.jpush.android.intent.DaemonService", "com.taobao.accs.intent.action.SERVICE", "com.taobao.accs.intent.action.ELECTION", "com.taobao.accs.intent.action.RECEIVE", "com.taobao.accs.intent.action.RECEIVE", "org.agoo.android.intent.action.RECEIVE", "org.agoo.android.intent.action.RECEIVE", "org.android.agoo.client.MessageReceiverService", "com.umeng.messge.registercallback.action", "com.umeng.message.enablecallback.action", "com.umeng.message.disablecallback.action", "com.umeng.message.message.handler.action", "com.umeng.message.message.sendmessage.action", "cn.jpush.android.intent.REGISTRATION", "cn.jpush.android.intent.MESSAGE_RECEIVED", "cn.jpush.android.intent.NOTIFICATION_RECEIVED", "cn.jpush.android.intent.NOTIFICATION_OPENED", "android.intent.action.BOOT_COMPLETED", "android.net.conn.CONNECTIVITY_CHANGE", "cn.jpush.android.intent.NOTIFICATION_RECEIVED_PROXY", "android.intent.action.USER_PRESENT", "android.net.conn.CONNECTIVITY_CHANGE", "android.intent.action.PACKAGE_ADDED", "android.intent.action.PACKAGE_REMOVED", "android.intent.action.BOOT_COMPLETED", "android.net.conn.CONNECTIVITY_CHANGE", "android.intent.action.PACKAGE_REMOVED", "android.intent.action.USER_PRESENT", "com.taobao.accs.intent.action.COMMAND", "com.taobao.accs.intent.action.START_FROM_AGOO", "com.hsbxjj.middleman.app.intent.action.COMMAND", "android.intent.action.PACKAGE_REMOVED", "com.huawei.android.push.intent.REGISTRATION", "com.huawei.android.push.intent.RECEIVE", "com.huawei.android.push.intent.CLICK", "com.huawei.intent.action.PUSH_STATE", "com.huawei.intent.action.PUSH", "com.meizu.cloud.pushservice.action.PUSH_SERVICE_START", "android.net.conn.CONNECTIVITY_CHANGE", "com.xiaomi.push.PING_TIMER", "com.xiaomi.mipush.RECEIVE_MESSAGE", "com.xiaomi.mipush.MESSAGE_ARRIVED", "com.xiaomi.mipush.ERROR"]